spec/controllers/logs_controller_spec.rb

```diff
@@ -40,7 +40,7 @@
     it "assigns the requested log as @log" do
       log = Log.create! valid_attributes
       get :show, {:id => log.to_param}, valid_session
-      expect(assigns(:log)).to eq(log)
+      expect(controller.instance_variable_get("@log")).to eq(log)
```

Comment:
`assigns` method is moved to rails-controller-testing gem.
But I failed to install it...
So, get variables by `instance_variable_get` method.